README.md

```diff
@@ -55,10 +144,10 @@ cdk init app --language python
 python -m venv .env
 
 # activate the virtualenv
-source env/bin/activate
+source .env/bin/activate
 
 # install the packages from requirements.txt
-pip install -r requirements.txt
+pip3 install -r requirements.txt
```

Comment:
I had issues with my local install if I didn't use `pip3` ... it may be a local path/alias issue.

Reply:
I have pip3 aliased as pip. It saves a lot of hassle.